src/main/java/co/rsk/federate/BtcToRskClient.java

```diff
@@ -115,25 +116,51 @@ public synchronized void setup(
  }
 
     public void start(Federation federation) {
-        logger.info("Starting for Federation {}", federation.getAddress().toString());
+        logger.info("[start] Starting for Federation {}", federation.getAddress());
         this.federation = federation;
-        if (federation.isMember(federatorSupport.getFederationMember())) {
-            logger.info("Watching federation {} since I belong to it", federation.getAddress().toString());
-            bitcoinWrapper.addFederationListener(federation, this);
-            int federatorIndex = federation.getBtcPublicKeyIndex(federatorSupport.getFederationMember().getBtcPublicKey());
-            TurnScheduler scheduler = new TurnScheduler(
-                    bridgeConstants.getUpdateBridgeExecutionPeriod(),
-                    federation.getSize()
-            );
-            long now = Clock.systemUTC().instant().toEpochMilli();
+        FederationMember federator = federatorSupport.getFederationMember();
+        boolean isMember = federation.isMember(federator);
+        
+        if (!isMember) {
+            logger.info("[start] member is no part of the federation {} ",
```

Comment:
log the

Reply:
it throws a NullPointerException since you cannot get a federator's public key from a non-federator

Reply:
You'll need to update a few FederationMember mocks in those tests to avoid the NPE. It think the message it's useful in the logs